Mathlib/Algebra/Homology/DifferentialObject.lean

```diff
@@ -93,6 +92,8 @@ def dgoToHomologicalComplex :
         -- Porting note: this `rw` used to be part of the `simp`.
         have : f.f i ≫ Y.d i = X.d i ≫ f.f _ := (congr_fun f.comm i).symm
         rw [reassoc_of% this] }
+  map_id _ := rfl
+  map_comp _ _ := rfl
```

Comment:
Proofs like this, which can be done by the auto_param, but are slow, need to have a comment added, or they will just get removed again.